### DISCUSSION TITLE
Zen30 json error "If" condition for parameter

### DISCUSSION THREAD

**Discussion by apella12** [zwave-js/zwave-js#6631] [General] [answered]
First time trying a complex expression and getting a Lint error with this expression.
`"$if": "firmwareVersion == 1.11 || firmwareVersion == 2.11 || firmwareVersion >= 3.10",`
Also tried
`"$if": "firmwareVersion = 1.11 || firmwareVersion = 2.11 || firmwareVersion >= 3.10",`

Message is ```
Error: Invalid logic: firmwareVersion == 1.11 || firmwareVersion == 2.11 || firmwareVersion >= 3.10
@zwave-js/config:lint:zwave: Expected grouped expression, logical AND, logical OR, numeric comparison, or version comparison but "f" found.
```

**Comment by AlCalzone** [comment, marked as answer]
Use triple equals `===` for equality. The syntax is JS-like.

**Comment by apella12** [comment]
Thanks !
